src/app/home-redesign/data.ts

```diff
@@ -0,0 +1,76 @@
+import { GenericResponseType } from '@/common/hooks/useInfiniteQueryResult';
+import { getApiUrl } from '@/common/utils/network-utils';
+
+import { Block, BurnBlock } from '@stacks/stacks-blockchain-api-types';
+
+import {
+  RECENT_BTC_BLOCKS_COUNT,
+  RECENT_STX_BLOCKS_COUNT,
+} from '../_components/RecentBlocks/consts';
+
+export type UIBtcBlock = Pick<
+  BurnBlock,
+  'burn_block_height' | 'burn_block_time' | 'burn_block_hash' | 'stacks_blocks' | 'total_tx_count'
+>;
+
+export type UIStxBlock = Pick<
```

Comment:
Just thinking out loud. I think this name is too general. I can see other situations in the explorer where you could want to use the same name for this kind of type. Would be nice to be more specific as this type only pertains to the data needed by the recent blocks UIs.

Reply:
I've seen `UI` used as a type name prefix to indicate that it's a UI-specific interface. Open to different prefixes.